rcl_logging_log4cxx/src/rcl_logging_log4cxx/rcl_logging_log4cxx.cpp

```diff
@@ -41,32 +44,19 @@ static const log4cxx::LoggerPtr get_logger(const char * name)
   return log4cxx::Logger::getLogger(name);
 }
 
-/* These are defined here to match the severity levels in rcl. They provide a consistent way for external logger
-    implementations to map between the incoming integer severity from ROS to the concept of DEBUG, INFO, WARN, ERROR,
-    and FATAL*/
-enum RC_LOGGING_LOG_SEVERITY
-{
-  RC_LOGGING_SEVERITY_UNSET = 0,  ///< The unset log level
-  RC_LOGGING_SEVERITY_DEBUG = 10,  ///< The debug log level
-  RC_LOGGING_SEVERITY_INFO = 20,  ///< The info log level
-  RC_LOGGING_SEVERITY_WARN = 30,  ///< The warn log level
-  RC_LOGGING_SEVERITY_ERROR = 40,  ///< The error log level
-  RC_LOGGING_SEVERITY_FATAL = 50,  ///< The fatal log level
```

Comment:
Were these used anywhere downstream?

Reply:
No, they are only used internally (especially because they were only defined in the `.cpp` file).  They are actually duplicates of the [RCUTILS_LOG_SEVERITY enum](https://github.com/ros2/rcutils/blob/3daf12617c2bfe7d6eb49c68e2710d7727ef2e6d/include/rcutils/logging.h#L164), which is why I wanted to get rid of them.

(also, we never used this logging implementation in a release)